presidio-anonymizer/presidio_anonymizer/anonymizer_engine.py

```diff
@@ -28,6 +33,7 @@ def anonymize(
             text: str,
             analyzer_results: List[RecognizerResult],
             operators: Optional[Dict[str, OperatorConfig]] = None,
+            conflict_resolution: Optional[ConflictResolutionStrategy] = None
```

Comment:
```suggestion
            conflict_resolution: Optional[ConflictResolutionStrategy] = ConflictResolutionStrategy. MERGE_SIMILAR_OR_CONTAINED
```
Thanks. My only suggestion is this. If we keep it as None, it might seem as if there is no conflict resolution happening. WDYT?

Reply:
Hi @omri374 
Thanks for the commit. I am extremely sorry for the misunderstanding. Actually, now I got the clear understanding what you were trying to say about third case. For now you can proceed with this or we can connect and discuss the third case as well, also on its documentation and some finding on llm integration.